test/chart.test.ts

```diff
@@ -39,26 +41,30 @@ describe("xlsx-wasm test", () => {
       });
     });
   
-    const chart = new Chart(ChartType.Line);
+    const chart = new Chart(ChartType.Stock);
     const chartFont = new ChartFont().setName("Meiryo UI");
     const chartDataLabel = new ChartDataLabel().setFont(chartFont).showValue().setPosition(ChartDataLabelPosition.Left);
```

Comment:
FYI: The refactoring caused the following code to crash:
```suggestion
    const chartDataLabel = new ChartDataLabel().setFont(chartFont);
    chartDataLabel.showValue().setPosition(ChartDataLabelPosition.Left);
```

I think this is acceptable, as the current code doesn't support it.

Reply:
`chart_format` and `chart_font` also cause a crash. Thank you for your comment.